### PR TITLE
Created Login page cleanup

### PR DESCRIPTION
**Pull request by stevenshuhyow**

### CODE REVIEW

File: server/auth/authController.js [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
-    // console.log('im in authController');
+
+  //clientID and clientSecret need to remain private
+  //we need to add this information to heroku
+  clientID: localServices.facebook.id,
+  clientSecret: localServices.facebook.secret,
+  callbackURL: localServices.facebook.callback
+
+   }, function(token, refreshToken, profile, done){
+    console.log(profile.id);
```
Comment (ekong2) [inline server/auth/authController.js]: no need for console logs

File: server/auth/authController.js [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
-          // Creates a new user which extracts the Facbook ID and username
-          // Facebook ID is referenced so that users are not able to sign up for more than one account
+          console.log('creating new user');
           db.User.create({
             username: profile.name.givenName,
             facebook_id: profile.id
           })
           .then(function(newUser){
-            // null value represents that there were no errors
+            console.log(newUser, '----------------------');
```
Comment (ekong2) [inline server/auth/authController.js]: ^

File: server/auth/authController.js [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
+
+   }, function(token, refreshToken, profile, done){
+    console.log(profile.id);
+
     db.User.find({where: {'facebook_id' : profile.id}})
       .then(function(user){
         if(!user){
-          // Creates a new user which extracts the Facbook ID and username
-          // Facebook ID is referenced so that users are not able to sign up for more than one account
+          console.log('creating new user');
```
Comment (ekong2) [inline server/auth/authController.js]: same here, replace with comments

File: server/auth/authController.js [hunk 4]
```diff
@@ -1,40 +1,44 @@
 var FacebookStrategy = require('passport-facebook').Strategy;
 var db = require('../db');
-var config = require('../config/environment');
```
Comment (dmsakamoto) [inline server/auth/authController.js:3]: config needs to be back here

File: server/auth/authController.js [hunk 5]
```diff
@@ -1,40 +1,45 @@
 var FacebookStrategy = require('passport-facebook').Strategy;
 var db = require('../db');
+var localServices = require('../config/environment/thirdPartyServices');
```
Comment (dmsakamoto) [inline server/auth/authController.js]: Delete

File: server/auth/authController.js [hunk 6]
```diff
@@ -1,40 +1,45 @@
...
 var config = require('../config/environment');
+// var client = require('./facebookAuthInfo.js');
```
Comment (dmsakamoto) [inline server/auth/authController.js]: Delete

File: server/auth/authController.js [hunk 7]
```diff
@@ -1,40 +1,45 @@
...
-          // Creates a new user which extracts the Facbook ID and username
-          // Facebook ID is referenced so that users are not able to sign up for more than one account
+          // console.log('creating new user');
           db.User.create({
             username: profile.name.givenName,
             facebook_id: profile.id
           })
           .then(function(newUser){
-            // null value represents that there were no errors
+            // console.log(newUser, '----------------------');
```
Comment (dmsakamoto) [inline server/auth/authController.js:31]: Delete

**Comment by dmsakamoto** [comment]
:+1:

**Comment by ekong2** [comment]
can you squash the commits?

**Pull request merged**